README.md

```diff
@@ -13,95 +13,60 @@
-1. [Golang](https://github.com/okteto/go-getting-started)
-1. [Java Gradle](https://github.com/okteto/java-gradle-getting-started)
-1. [Java Maven](https://github.com/okteto/java-maven-getting-started)
-1. [Node](https://github.com/okteto/node-getting-started)
-1. [PHP](https://github.com/okteto/php-getting-started)
-1. [Python](https://github.com/okteto/python-getting-started)
-1. [Ruby](https://github.com/okteto/ruby-getting-started)
-
-More samples on how to use Okteto [are available here](https://github.com/okteto/samples).
-
-## Features
-
-### Development environments on demand 
-
-Your development environment is defined in a [simple yaml manifest](https://okteto.com/docs/reference/manifest).
-- Run `okteto init` to inspect your project and generate your own config file.
-- Run `okteto up` to launch your development environment in seconds. 
+## Why Okteto
 
-Add `okteto.yml` to your repo and make collaboration easier than ever. Clone the repository and simply run `okteto up` to launch a fully configured development environment.
+Okteto has several advantages versus traditional development:
+- **Fast inner loop development**: build and run with your favorite tools from your development container. Native builds are always faster than the *docker build/redeploy* cycle.
+- **Production-like environment**: your development container reuses the same variables, secrets, sidecars, volumes... than your original Kubernetes deployment. Realistic environments eliminate integration issues.
+- **Unlimited resources**: get access to the hardware and network of your cluster from development.
 
-### Developer Mode 
-
-You can swap your development environment with an existing Kubernetes deployment, and develop directly in your cluster. This helps eliminate integration issues since you're developing the same way your application runs in production.
-
-Okteto supports applications with one or with multiple services.
-
-### Instant file synchronization
-
-Okteto detects your code changes and synchronizes your code to your development environment (powered by [Syncthing](https://github.com/syncthing/syncthing)). Files are synchronized both ways. If you edit a file directly in your development environment, the changes will be reflected locally as well. Great for keeping your `package-lock.json` or `requirements.txt` up to date.
-
-### Keep Your Own Tools
+## Getting started
 
-Okteto easily integrates and augments your existing tools. You write code locally, which means Okteto is compatible with any IDE. And you can configure your own development environment runtime using a docker image, with your own compilers, linters, debuggers or hot reloaders.
+All you need to get started is to [install the Okteto CLI](https://okteto.com/docs/getting-started/installation/index.html) and have access to a Kubernetes cluster. You can also use Okteto in [Okteto Cloud](https://okteto.com/), the best development platform for Kubernetes applications.
```

Comment:
`the best development platform for Kubernetes applications`, I know this was already there but it still sounds odd to me. Maybe something like `our hosted development platform for Kubernetes applications` or `our approach to a hosted development platform for Kubernetes applications on top of Okteto`.

Reply:
I think it's better to remove that last sentence.